sdk/ai/azure-ai-resources/azure/ai/resources/client/_ai_client.py

```diff
@@ -309,18 +309,18 @@ def build_index_on_cloud(
         Returns:
             _type_: _description_
         """
-        from azure.ai.resources._index._dataindex.data_index import index_data
-        from azure.ai.resources._index._dataindex.entities import (
+        from azure.ai.generative.index._dataindex.data_index import index_data
```

Comment:
Can we keep classes ai-resources package needs in the resources package? Otherwise we may introduce a circular dependency between ai-resourcs and ai-generative packages.